packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -115,11 +143,11 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
 
     const submitHandler = (values, { setSubmitting, setErrors }) => {
         setSubmitting(true);
-        const { document_number, document_type } = values;
         const submit_data = {
             identity_verification_document_add: 1,
-            document_number,
-            document_type: document_type.id,
+            document_number: values.document_number,
+            document_additional: values.document_additional || '',
```

Comment:
@suisin-deriv is it okay if we send this extra key "document_additional" for all IDV documents?

Reply:
@amina-deriv I tried it can passed if i pass in extra key

Reply:
Thanks for the clarification @suisin-deriv 👌